lib/acmesmith/post_issueing_hooks/shell.rb

```diff
@@ -0,0 +1,36 @@
+require 'open3'
+require 'acmesmith/post_issueing_hooks/base'
+
+module Acmesmith
+  module PostIssueingHooks
+    class Shell < Base
+      class HostedZoneNotFound < StandardError; end
+      class AmbiguousHostedZones < StandardError; end
+
+      def initialize(common_name:, command:, ignore_failure:false)
+        @common_name = common_name
+        @command = command
+        @ignore_failure = ignore_failure
+      end
+
+      def execute
+        puts "=> Executing Post Issueing Hook for #{@common_name} in #{self.class.name}"
+        puts "=> ENV: COMMON_NAME=#{@common_name}"
+        puts "=> Running: #{@command}"
+
+        stdout, stderr, status = Open3.capture3({"COMMON_NAME" => @common_name}, "#{@command};")
```

Comment:
Ok, it should be ```_, stderr, _ = Open3.capture3({"COMMON_NAME" => @common_name}, "#{@command};")```
Capture will always return those 3.

I think you prefer to use ```system()```? I implemented de Open3 alternative because you requested $stderr output, but it seems not the weight as heavy as having little dependencies.